src/Pdp/PublicSuffixListManager.php

```diff
@@ -69,18 +70,28 @@ public function refreshPublicSuffixList()
         $publicSuffixListArray = $this->parseListToArray(
             $this->cacheDir . '/' . self::PDP_PSL_TEXT_FILE
         );
-        $this->writePhpCache($publicSuffixListArray);
+
+        // do not empty existing PHP cache file if source TXT is empty
+        if (is_array($publicSuffixListArray) && !empty($publicSuffixListArray)) {
```

Comment:
But if I run method `updateCache()` after it execution I want be sure that it was updated. Or I want receive exception that some error occured, with this information I can fix the problem.